.pipelines/cni/singletenancy/windows-cni-load-test-template.yaml

```diff
@@ -80,8 +86,27 @@ stages:
               kubectl get pods -A
             name: "WaitForCNI"
             displayName: "Wait For CNI"
-  - stage: pod_deployment_windows
+  - stage: datapath_tests
+    pool:
+      name: $(BUILD_POOL_NAME_DEFAULT)
     dependsOn: update_cni
+    displayName: "Datapath Test - Windows"
+    jobs:
+      - template: ../k8s-e2e/k8s-e2e-job-template.yaml
+        parameters:
+          sub: $(TEST_SUB_SERVICE_CONNECTION)

```

Comment:
Do we need to send the `sub` as a parameter to the templates ?

Reply:
Yes, some subscriptions have access to certain clusters.